src/components/AgreementSections/LinesList.js

```diff
@@ -64,7 +64,7 @@ export default class LinesList extends React.Component {
     activeFrom: line => <div data-test-active-from>{this.renderDate(line.startDate)}</div>,
     activeTo: line => <div data-test-active-to>{this.renderDate(line.endDate)}</div>,
     count: line => <EResourceCount resource={getResourceFromEntitlement(line)} />,
-    coverage: line => <CoverageStatements statements={line.coverage} />,
+    coverage: line => <Coverage pci={line?.resource?._object} />,
```

Comment:
Are you sure about this? It looks like you're now showing the base coverage and not custom coverage if it exists.